docs/includes/metrics/micrometer-shared.adoc

```diff
@@ -90,22 +87,39 @@ For example, Helidon configures the built-in Prometheus registry using `Promethe
 To use configuration to control the selection and behavior of Helidon's built-in Micrometer meter registries,
 include in your configuration (such as `application.yaml`) a `micrometer.builtin-registries` section.
 
-[source,yaml]
 .Enroll Prometheus built-in meter registry using default configuration
+ifdef::se-flavor[]
+[source,yaml]
 ----
 micrometer:
   builtin-registries:
     - type: prometheus
 ----
+endif::[]
+ifdef::mp-flavor[]
+[source,properties]
+----
+micrometer.builtin-registries.0.type=prometheus
+----
+endif::[]
 
-[source,yaml]
 .Enroll Prometheus built-in meter registry with non-default configuration
+ifdef::se-flavor[]
+[source,yaml]
 ----
 micrometer:
   builtin-registries:
     - type: prometheus
       prefix: myPrefix
 ----
+endif::[]
+ifdef::mp-flavor[]
+[source,properties]
+----
+micrometer.builtin-registries.0.type=prometheus
+micrometer.builtin-registries.0.prefix=myPrefix
+----
+endif::[]
 Note that the first config example is equivalent to the default Helidon Micrometer behavior; Helidon by default supports the Prometheus meter registry.
 
 The configuration keys that are valid for the `builtin-registries` child entries depend on the type of Micrometer meter
```

Comment:
provide a link to the documentation?

Reply:
Added a link to the Micrometer Prometheus registry config interface.